CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## Unreleased
+
+### Added
+- Migrate API translation code to plugin infra
+- Support for self transpilation with improved error reporting
+- New target: python. Transpiles untyped python to typed python.
+- Directory mode. Cross module type inference.
+- Experimental: pyO3 extension for rust
```

Comment:
Perhaps pyO3 mode could be converted into a new target "pyO3" or "rust+pyO3" ?

Reply:
It's a sub-target of rust, not a standalone one. A large module with a small interface to python would be mostly the same as rust. Only the code that interfaces with python would be transpiled differently.